src/colorizer/utils/react_utils.ts

```diff
@@ -413,50 +439,71 @@ export const useAnnotations = (): AnnotationState => {
     [lastEditedRange, lastClickedId]
   );
 
+  const nextDefaultLabelValue = useMemo(() => {
+    if (currentLabelIdx === null) {
+      return null;
+    }
+    return annotationData.getNextDefaultLabelValue(currentLabelIdx, isReuseValueHotkeyPressed);
+  }, [currentLabelIdx, dataUpdateCounter, isReuseValueHotkeyPressed]);
```

Comment:
The next value that will be applied is shown in the canvas hover tooltip.

![image](https://github.com/user-attachments/assets/dd0b2bcb-769d-44cb-876e-edd608005cc5)